assets/v2/cosmos/assets.json

```diff
@@ -650,5 +650,43 @@
     },
     "image": "common/scrt.png",
     "coinGeckoId": "secret"
+  },
+  {
+    "denom": "ibc/DB9AAADFBE21E014373EF97141D451F453B9B9A2C8B65F5CF8E7CE57531B6850",
+    "type": "ibc",
+    "base_denom": "basecro",
```

Comment:
@wannabit-yongjoo Maybe I have to change base_denom to transfer/channel-31/basecro and origin_chain to evmos as the asset is not visualized correctly? eg. https://www.mintscan.io/cosmos/txs/779CFB9ABDCAFB6C1DA0C24F8CEF1B977A3814B2DBA3A2014D636B847285E944